sdk/translation/azure-ai-translation-text/src/main/java/com/azure/ai/translation/text/TextTranslationAsyncClient.java

```diff
@@ -709,26 +550,116 @@ public Mono<List<TranslatedTextItem>> translate(List<String> to, List<InputTextI
+     * For example, use to=de&amp;to=it to translate to German and Italian.
+     * @param text Defines the content of the request.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return the response.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<List<TranslatedTextItem>> translate(String targetLanguages, String text) {
+        return translate(targetLanguages, Arrays.asList(text));
+    }
+
+    /**
+     * Translate Text.
+     *
+     * @param text Text to translate.
+     * @param translateOptions Translate Options.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return the response.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<List<TranslatedTextItem>> translate(String text, TranslateOptions translateOptions) {
+        return translate(Arrays.asList(text), translateOptions);
+    }
+
+    /**
+     * Translate Text.
+     *
+     * @param texts List of text to translate.
+     * @param translateOptions Translate Options.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return the response.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<List<TranslatedTextItem>> translate(List<String> texts, TranslateOptions translateOptions) {
+        ArrayList<InputTextItem> content = new ArrayList<>();
```

Comment:
nit: use the `List` interface 
```suggestion
        List<InputTextItem> content = new ArrayList<>();
```

Reply:
Changed.